python/lsst/dax/distribution/DataGenServer.py

```diff
@@ -474,14 +487,18 @@ def start(self):
         print("starting")
         self._servAccept()
         print("Done, generated ", self._total_generated_chunks)
+
         print("chunks failed chunks:", self.chunksInState([GenerationStage.LIMBO, GenerationStage.ASSIGNED]))
         counts = {GenerationStage.UNASSIGNED:0,
-            GenerationStage.ASSIGNED:0,
-            GenerationStage.FINISHED:0,
-            GenerationStage.LIMBO:0}
+                  GenerationStage.ASSIGNED:0,
+                  GenerationStage.FINISHED:0,
+                  GenerationStage.LIMBO:0}
```

Comment:
What's the advantage of using enum here vs a simple Dict for counts?

Reply:
Mostly, it gives runtime errors if one of the values is misspelled.